### DISCUSSION TITLE
Serving jpg for social images on LinkedIn

### DISCUSSION THREAD

**Discussion by Harold-Anderson** [arthelokyo/astrowind#471] [Q&A]
It seems that Astrowind is transforming blog post images to webp, which LinkedIn does not accept.  Is there a way to specify an alternate social media image or to turn off the transformation to webp for blog images?

**Comment by Sloopdog** [comment]
Are you referring to the the image "cover image" at the top of the post? If so you can use a jpg pic in your public folder and reference that. When images are in the public folder Astro does not optimize those, so it wont be converted to webp

**Comment by Harold-Anderson** [comment]
The behavior does not agree with the documentation.  The documentation states that Astro will not optimize pictures in /public as you say, but in reality they are optimized.  the workaround is to x-out the picture in LinkedIn and upload a jpeg before you hit Post.  @Sloopdog
